content/en/platform/corda/4.8/enterprise/docker-image.md

```diff
@@ -11,175 +11,23 @@ weight: 120
 ---
 
 
-# Official Corda Docker image
+# Official Corda Docker Image
```

Comment:
I believe this is a lower case I on image. Sentence case for titles.

Reply:
If not then Docker image and Docker Image are used inconsistently here so pls check this.